external.bzl

```diff
@@ -1,12 +1,12 @@
+load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-load("@io_kythe//tools:build_rules/shims.bzl", "go_repository")
-load("@io_kythe//:setup.bzl", "maybe")
+load("@build_bazel_rules_nodejs//:package.bzl", "rules_nodejs_dependencies")
 
 # Rule dependencies
```

Comment:
Does this comment mean anything useful? I notice the tool permuted some loads around the comment, which suggests it's either meaningless or that possibly it has become incorrect.